### ISSUE TITLE
Error: failed to upload distribution. it took longer than expected to process your binary, please try again

### ISSUE THREAD

**Issue by YkmLo** [firebase/firebase-tools#2157]
### [REQUIRED] Environment info

**firebase-tools:** 8.1.1

**Platform:** macOS

### [REQUIRED] Test case

Uploading via fastlane x firebase CLI

### [REQUIRED] Steps to reproduce

``` ruby
firebase_app_distribution(
      app: app_id,
      firebase_cli_token: token,
      ipa_path: ipaPath,
      release_notes: changelogs,
      debug: true
)
```

filesize: 253M

### [REQUIRED] Expected behavior

Success upload

### [REQUIRED] Actual behavior

last output:

> HTTP REQUEST GET https://firebaseappdistribution.googleapis.com/v1alpha/apps/{redacted}/upload_status/{redacted}

> <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Wed, 22 Apr 2020 09:13:59 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"quic=\":443\"; ma=2592000; v=\"46,43\",h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}

> ▸ Error: failed to upload distribution. it took longer than expected to process your binary, please try again

**Comment by tonybaroneee** [comment]
Hey @YkmLo, thanks for the report. We’re currently in the process of improving the binary upload speed to resolve this issue that some customers in various parts of the world are seeing. We’ll let you know when that’s live!

**Comment by danwilsonlg** [comment]
Hi Firebase team! Can you confirm that you're still working on this? Members of our engineering team have had to drive to places with really fast wifi to upload builds without timing out. (Actual chat snippet: "Android ready to test! Driving to Walmart to get iOS ready.")

**Comment by kaibolay** [comment]
This should be fixed with with #2251 which was just released as part of https://github.com/firebase/firebase-tools/releases/tag/v8.4.1

Please let us know if it fixes your problem.

**Comment by YkmLo** [comment]
wow, thanks guys. yup it's fixed :D
